cmd/web/web.go

```diff
@@ -136,6 +144,18 @@ func main() {
 	// For testing purpose
 	// http.HandleFunc("/gh/test", handleTest)
 
+	// Initialize chan
+	cacheJobs = make(chan string, 100) // maximum number of jobs that can be queued
+	go checkMemoryCache(cacheJobs) // start coroutine
+
+	// Initialize cache
+	processMemoryCache(cacheTypeSummary)
+	processMemoryCache(cacheTypeTopStar)
+	if len(cacheSummary) == 0 || len(cacheTopStar) == 0 {
+		log.Fatal("Failed to initialize in-memory cache!")
```

Comment:
log.Fatal will implicitly does os.Exit right?

Reply:
oh but I think this one is expected, so it's okay

Reply:
Ah yes, the application should not start if it fails on the beginning (the `return` is a bit unnecessary though).